proto/firecracker.proto

```diff
@@ -36,6 +36,15 @@ message CreateVMRequest {
     JailerConfig JailerConfig = 10;
 }
 
+message CreateVMResponse {
+    string VMID = 1;
+    uint32 ContextID = 2;
```

Comment:
nit: `ContextID` is no longer set or used anywhere after the recent vsock changes, we may as well remove it from both CreateVMResp and GetVMInfoResp

Reply:
Done!